src/darwin/Framework/CHIP/MTRCluster.h

```diff
@@ -61,8 +61,7 @@ NS_ASSUME_NONNULL_BEGIN
  * from the sever to the client (for the status response and actual write
  * request) within the timeout window.
  *
- * This value is specified in milliseconds
- *
+ * This value is specified in milliseconds.
```

Comment:
Further down it seems we have timeouts in seconds? It's a little unfortunate to have inconsistent time units.. The usual type to use would be `NSTimeInterval` i.e. seconds.

Reply:
The problem is that these are very different kinds of timeouts....  NSTimeInterval allows non-integer seconds, right?

I'll look into whether we can switch to NSTimeInterval as a followup.